### PR TITLE
Move actions definition to ExportActionMixin

### PR DESCRIPTION
**Pull request by lorenzomorandini**
New way of handling admin actions

**Problem**

From Django 2.2 admin actions are no longer collected from base ModelAdmin classes
https://docs.djangoproject.com/en/2.2/releases/2.2/#admin-actions-are-no-longer-collected-from-base-modeladmin-classes

**Solution**

Use new suggested way of adding admin actions

**Acceptance Criteria**

Have you written tests? Have you included screenshots of your changes if applicable?
Did you document your changes?

### CODE REVIEW

**Comment by lorenzomorandini** [comment]
Fix #947

File: import_export/admin.py [hunk 1]
```diff
@@ -577,6 +575,8 @@ class ExportActionModelAdmin(ExportActionMixin, admin.ModelAdmin):
     admin action.
     """
 
+    actions = admin.ModelAdmin.actions + [export_admin_action]
```
Comment (andrewgy8) [inline import_export/admin.py]: I think `actions = admin.ModelAdmin.actions + [ExportActionMixin.export_admin_action]` should fix your broken tests.

**Comment by andrewgy8** [comment]
Hey @lorenzomorandini ! Thanks for the fix. Lets get those tests passing and merge it!

**Comment by andrewgy8** [comment]
Thanks @lorenzomorandini !

**Pull request merged**